base_layer/wallet/src/test_utils.rs

```diff
@@ -0,0 +1,253 @@
+// Copyright 2019. The Tari Project
```

Comment:
call this testnet_utils? And tTo make it clear that it's not test code, please elaborate on the mod comment below

Reply:
Changed the name and added a more explicit comment at the top of the file. In my next PR I hide those whole file behind a feature flag to make it even clearer.